### DISCUSSION TITLE
Make it easier to select a connection? (v10)

### DISCUSSION THREAD

**Discussion by raarts** [xyflow/xyflow#1934] [Q&A] [answered]
I find it pretty hard to select an edge, because the lines are only 1 pixel wide. Is there a way to make selecting easier?

**Comment by moklick** [comment, marked as answer]
Hey @raarts 

you can make the edges thicker with help of the `stroke-width` attribute or use an underlying thick ghost edge (explained here: https://github.com/wbkd/react-flow/issues/1114#issuecomment-824600239).
